nise/upload.py

```diff
@@ -18,7 +18,7 @@
 import os
 
 import boto3
-from azure.storage.blob import BlockBlobService
+from azure.storage.blob import BlobServiceClient
```

Comment:
Unpinning the Azure deps included an API-breaking change. `BlockBlobService` is the old API (pre-2018) and `BlobServiceClient` is the new API (post-2019). So, I've updated the interface as part of this change.